packages/graph-explorer/src/connector/openCypher/templates/keywordSearchTemplate.test.ts

```diff
@@ -2,6 +2,17 @@ import { normalize } from "@/utils/testing";
 import keywordSearchTemplate from "./keywordSearchTemplate";
 
 describe("OpenCypher > keywordSearchTemplate", () => {
+  it("Should return a template for empty request", () => {
```

Comment:
Should there be an equivalent test to this for gremlin?

Reply:
It already exists, albeit with a slightly bad name:

https://github.com/aws/graph-explorer/blob/e259daf0f14fff4dac8f568096d12934a9e5e4b9/packages/graph-explorer/src/connector/gremlin/templates/keywordSearchTemplate.test.ts#L5-L9

Reply:
I've updated the name since that original name made no sense.